cms/envs/common.py

```diff
@@ -47,7 +47,8 @@
     update_module_store_settings, ASSET_IGNORE_REGEX, COPYRIGHT_YEAR,
     # The following setting is included as it is used to check whether to
     # display credit eligibility table on the CMS or not.
-    ENABLE_CREDIT_ELIGIBILITY, YOUTUBE_API_KEY
+    ENABLE_CREDIT_ELIGIBILITY, YOUTUBE_API_KEY,
+    DEFAULT_COURSE_ABOUT_IMAGE_URL,
```

Comment:
Alternatively, if you add this further up, you won't have to add the trailing comma and this will be less likely to conflict down the line.